src/main/resources/themes/one_dark.theme.json

```diff
@@ -312,8 +305,7 @@
 
       "Header": {
         "background": "#414855",
-        "inactiveBackground": "#323844",
-        "borderColor": "#21252b"
+        "inactiveBackground": "#21252b"
```

Comment:
Why this change?

Reply:
Probably this one from tweaking around will revert it

Reply:
Reverted